Examples/HttpAnalyzer/main.cpp

```diff
@@ -582,10 +582,6 @@ int main(int argc, char* argv[])
 	{
 		EXIT_WITH_ERROR("Please input a number between 0 to 65535");
 	}
-	if (nPort < 0 || nPort > 65535)

```

Comment:
I think there is a bug here, not in your fix but in the previous code: if `port` represents a number that is larger than 65535 or smaller than 0 then `nPort == 65535`. Also if `port` is not a number then `nPort == 0`. Maybe we want to fix this instead of removing these lines?